Documents/TechnicalSpecifications/technicalSpecifications.md

```diff
@@ -2,32 +2,69 @@
 
 ## Table of content
 
-- [Functional Specifications](#functional-specifications)
+- [Technical specifications](#technical-specifications)
   - [Table of content](#table-of-content)
+  - [Document purpose](#document-purpose)
 - [Project definition](#project-definition)
   - [Overview](#overview)
+  - [Team members](#team-members)
   - [Description](#description)
 - [Development environment](#development-environment)
   - [Hardware](#hardware)
   - [Programming language](#programming-language)
-  - [Conventions](Conventions)
-    - [File tree and documents](#file-tree-and-document)
+  - [Conventions](#conventions)
+    - [File tree and documents](#file-tree-and-documents)
     - [Code](#code)
       - [Naming conventions](#naming-conventions)
       - [Structure conventions](#structure-conventions)
-
+- [Software architecture](#software-architecture)
+  - [Game actions flow](#game-actions-flow)
+  - [Software file management](#software-file-management)
+    - [File tree](#file-tree)
+    - [Files interactions](#files-interactions)
+  - [Program structure](#program-structure)
+- [Risks and mitigations strategy](#risks-and-mitigation-strategies)
+  - [Risks](#risks)
+  - [Mitigation strategies](#mitigation-strategies)
 - [Glossary](#glossary)
 
 
+## Document purpose
+
+The purpose of this document is to describe how the program will be created. It will be updated following the advancements of the project, to match what as been done during the development phase.
+
 # Project definition
 
 ## Overview
 
+The project, given by ALGOSUP, is to create a serious game about climate change<sup><a href="#1">[1]</a></sup>. It has to teach what is creating the climate change and how to face it. More details about the project requirements and design are available in the [functional requirements documentation](../FunctionalSpecifications/functionalSpecifications.md).
 
+## Team members
+
+| Name | Role |
+| -- | -- |
+| Mathias GAGNEPAIN | Project manager |
+| Enzo GUILLOUCHE | Program manager |
+| Evan UHRING | Technical leader |
+| Michel RIFF | Software engineer |
+| Loïc NOGUES | Software engineer |
+| Axel DAVID | Tech writer |
+| Tino GABET | Quality assurance |
 
 ## Description
 
+The game goal is to raise awareness about the climate change that the humanity is currently experiencing. To do so, here are the client requirements and our set objectives:
 
+| Client requirements |
+| -- |
+| Create a serious game |
+| Raise awareness about climate change |
+| Use the Godot game engine |
+
+| Our objectives |
+| -- |
+| Use 2D graphics (exterior: top-down view, interior: side view) |
```

Comment:
Be careful we use a 3D environment, you should mention it and explain why this choice in a 2D game

Reply:
done 👍